clients/src/main/java/org/apache/kafka/clients/admin/KafkaAdminClient.java

```diff
@@ -1122,10 +1108,10 @@ void handleFailure(Throwable throwable) {
 
     @Override
     public ListTopicsResult listTopics(final ListTopicsOptions options) {
-        final KafkaFutureImpl<Map<String, TopicListing>> topicListingFuture = new KafkaFutureImpl<>();
+        final KafkaFutureImpl<Map<String, TopicListItem>> topicListingFuture = new KafkaFutureImpl<>();
```

Comment:
I don't like `TopicListItem`, but couldn't think of a better name. I was looking for the word `Listing` in the dictionary and it seemed to mean something else, e.g.:

```text
1. A list or catalogue.
```

But looking deeper, there is something that could be what you're thinking of:

```text
1.2 An entry in a list or register.
```

Is that right?